Script/char.dat

```diff
@@ -3031,7 +3532,68 @@ femaleslave
     {
       2,
       "\"Never say no to Petrus. Ever!\"",
-      "\"And I thought Richel Decos was bad with his whips...\"";
+      "\"And I thought Richel Decos was bad with his whips...\"";
```

Comment:
It does add to the game, because it paints the brutal police state of Attnam and the unlimited power and nonexistent morals Petrus enjoys. It also adds prisoners into the dungeon, which makes it much more lively and interesting. So it does develop characters and narrative, but not comedy. It's not supposed to be comedic.

I understand you dislike victimized women, and I understand the lobotomy was too much. But there is also the imprisoned farmer who definitely didn't ask for being castrated, plus I try to balance it by better female NPCs - just see the new Petrus' wives or lady Decora.

The number of lines is low, but still on par with many other NPCs. Not everybody needs so many lines as the bum to tell their story. :P